ingress/loadbalancer_util.go

```diff
@@ -0,0 +1,39 @@
+package ingress
+
+import (
+	log "github.com/Sirupsen/logrus"
+)
+
+// Functions that benefit from isolated testing
+
+// ValidateEntry returns whether the given entry is valid
+func ValidateEntry(entry LoadBalancerEntry) bool {
+	if entry.Host == "" {
+		return false
+	}
+	if entry.Path == "" {
+		return false
+	}
+	if entry.ServiceName == "" {
+		return false
+	}
+	if entry.ServicePort == 0 {
+		return false
+	}
+	return true
+}
```

Comment:
Can this be a method on the LoadBalancerEntry itself? So we can do `entry.Validate()`?